src/index.js

```diff
@@ -209,16 +209,27 @@ function pluginTester({
 const createFixtureTests = (fixturesDir, options) => {
   fs.readdirSync(fixturesDir).forEach(caseName => {
     const fixtureDir = path.join(fixturesDir, caseName)
+    const optionsPath = path.join(fixtureDir, 'options.json')
     const jsCodePath = path.join(fixtureDir, 'code.js')
     const tsCodePath = path.join(fixtureDir, 'code.ts')
     const blockTitle = caseName.split('-').join(' ')
     const codePath =
       (pathExists.sync(jsCodePath) && jsCodePath) ||
       (pathExists.sync(tsCodePath) && tsCodePath)
 
+    let fixturePluginOptions = {}
+    if (pathExists.sync(optionsPath)) {
+      fixturePluginOptions = JSON.parse(
+        fs.readFileSync(optionsPath, 'utf8').toString(),
```

Comment:
Any reason we're not just using `require(optionsPath)` here?

Reply:
Not really, is that preferred?

Reply:
Meh, I'm fine either way. I just think require looks nicer.